frontend/src/cmps/offer-list.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+<section class="offer-list-container" v-if="offers">
+  <h1>Offer List</h1>
+  <div class="offers-topbar flex space-between">
+      <span>Product</span>
+      <span>Dates</span>
+      <span>Status</span>
+  </div>
+  
+    <offer-preview
+    v-for="offer in offers"
+    :key="offer._id"
+    :offer="offer" />
+ 
+</section>
+</template>
+
+<script>
+import offerPreview from './offer-preview.vue';
+import offerService from '../services/offer.service.js';
+export default {
+    name: 'offer-list',
+    data(){
+        return{
+            offers: null
+        }
+    },
+    created(){
+        this.loadOffers()
+    },
+    methods:{
+        async loadOffers(){
+           await this.$store.dispatch('loadOffers');
```

Comment:
i think it should be like that:
```suggestion
const offers =  await this.$store.dispatch('loadOffers');
```
and then you don't need to call the getter of the store after you call the dispatch for loading offers, because the dispatch will return the loaded users. am i right? (we have example for that in toys store: 
```suggestion
loadToys(context, payload) {
            const filter = context.state.filter;
            return toyService.query(filter)
                .then(toys => {
                    context.commit({
                        type: 'setToys',
                        toys
                    });
                    return toys;
                });
```